tensorflow/compiler/xla/service/gpu/nvptx_compiler.h

```diff
@@ -38,56 +39,29 @@ limitations under the License.
 namespace xla {
 namespace gpu {
 
-// Temporarily expose the optimization pipeline for the GPU backend for reuse
-// in the MLIR GPU backend.
-// TODO(b/137624192): Remove once MLIR backend uses tailored optimizations.
-namespace impl {
-
-Status OptimizeHloModule(HloModule* hlo_module, se::StreamExecutor* stream_exec,
-                         se::DeviceMemoryAllocator* device_allocator);
-Status PrepareHloModuleForIrEmitting(HloModule* hlo_module);
-
-}  // namespace impl
-
-// The GPU compiler generates efficient GPU executables.
-class NVPTXCompiler : public LLVMCompiler {
+// NVPTXCompiler generates efficient GPU executables for NVPTX target.
+class NVPTXCompiler : public GpuCompiler {
  public:
-  NVPTXCompiler();
+  NVPTXCompiler(se::Platform::Id platform_id);
```

Comment:
We always want stream_executor::cuda::kCudaPlatformId, right? Please hardcode it here. This way, callers do not need to be changed.

Reply:
will do.